packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -744,6 +749,15 @@ class RenderParagraph extends RenderBox
     return _textPainter.getOffsetForCaret(position, caretPrototype);
   }
 
+  /// Returns the tight bounded height of the glyph at the given [position].
```

Comment:
tight -> strut

Can you change this in the original docs in TextPainter as well? This doc is no longer accurate.

Reply:
Going off of the macro comment by Justin, change this in the original doc, and then macro it here.

Reply:
Updated and added a macro in TextPainter. I haven't included the last line referencing `[layout]` of TextPainter as it seemed specific to the painter class itself (even though RenderParagraph has a similar requirement).

Let me know if you'd like that line to be included in the template, I'll update.

Also changed `[position]` to use backticks, as it seems more inline with the style guide around method parameters.